### DISCUSSION TITLE
can get streetcomplete quest in other editor?

### DISCUSSION THREAD

**Discussion by Monirzadeh** [streetcomplete/StreetComplete#4799] [Q&A]
Hi
is it possible to get streetcomplete quest inside other editor like JOSM?

**Comment by matkoniecz** [comment]
Not aware about any such tools, but it would be relatively feasible to make JOSM map style or custom validators rules highlighting missing data detected by StreetComplete (with some exceptions).

Putting StreetComplete interface into JOSM would be neither feasible nor a good idea.

**Comment by matkoniecz** [comment]
https://github.com/bryceco/GoMap (Apple-only editor) is now using StreetComplete filters.

**Comment by letypequividelespoubelles** [comment]
I would love to have StreetComplete quests and overlay in OsmAnd, I wouldn't have to switch between the two apps while mapping !

**Reply by mnalis** [reply]
Well, You should propose that in [OsmAnd issue tracker](https://github.com/osmandapp/OsmAnd/issues), then... 

BTW if you are using Android, it allows to split screen in half and have both applications running at the same time, each having a half of the screen; so that solution might work for you too.
